prisma/schema.prisma

```diff
@@ -35,6 +35,7 @@ model ResourceOwner {
   Code           Code[]
   authorizedApps AuthorisedApps[]
   Token          Token[]
+  batch          String           @default("NA")
```

Comment:
can we change this to Enum like `BTech First Year`, `BTech Second Year`, if we need this data for proper things

or else we can go with the old one, @default can be `NULL`